### PR TITLE
What is Cibyl + readthedocs folder (2 commits)

### PR DESCRIPTION
**Pull request by dsariel**
- What is Cibyl?
- Add readthedocs folder

### CODE REVIEW

**Comment by adrianfusco** [comment]
@dsariel just a doubt, the `conf.py` file will be in another PR?

```
Running Sphinx v4.4.0

Configuration error:
config directory doesn't contain a conf.py file (/.../cibyl/docs/source)
make: *** [Makefile:20: html] Error 2
```

Thx.

**Review by bregman-arie — CHANGES_REQUESTED** [review]

File: docs/source/contribute.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Contribute
+==========
+
+Please isubmit a pull requests to the `cibyl project`_ on GitHub.
+
+.. _`cibyl project`: https://github.com/redhat-openstack/cibyl
```
[the remark this answers is not in the document]
Comment (dsariel) [reply docs/source/contribute.rst:6]: Done

File: docs/source/conf.py [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+# source_suffix = ['.rst', '.md']
+source_suffix = '.rst'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = u'infrared'
+copyright = u'2016, Red Hat'
+author = u'Yair Fried'
```
Comment (bregman-arie) [inline docs/source/conf.py:59]: Yair Fried? :'(

File: docs/source/conf.py [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+# Add any paths that contain custom static files (such as style sheets) here,
+# relative to this directory. They are copied after the builtin static files,
+# so a file named "default.css" will overwrite the builtin "default.css".
+html_static_path = ['_static']
+
+
+# -- Options for HTMLHelp output ------------------------------------------
+
+# Output file base name for HTML help builder.
+htmlhelp_basename = 'infrareddoc'
```
Comment (bregman-arie) [inline docs/source/conf.py:110]: infrared

File: docs/source/conf.py [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+    #
+    # 'figure_align': 'htbp',
+}
+
+# Grouping the document tree into LaTeX files. List of tuples
+# (source start file, target name, title,
+#  author, documentclass [howto, manual, or own class]).
+latex_documents = [
+    (master_doc, 'infrared.tex', u'infrared Documentation',
+     u'Yair Fried', 'manual'),
```
Comment (bregman-arie) [inline docs/source/conf.py:138]: these lines as well

File: docs/source/conf.py [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+]
+
+
+# -- Options for Texinfo output -------------------------------------------
+
+# Grouping the document tree into Texinfo files. List of tuples
+# (source start file, target name, title, author,
+#  dir menu entry, description, category)
+texinfo_documents = [
+    (master_doc, 'infrared', u'infrared Documentation',
```
Comment (bregman-arie) [inline docs/source/conf.py:158]: infrared

**Review by bregman-arie — CHANGES_REQUESTED** [review]

**Comment by bregman-arie** [comment]
This is open for 3 weeks. Let's close it in the meantime to not see it in the queue. We can revisit it once it's updated.

**Pull request closed without merging**